LokiLabs/microsoft-graph-explorer-v4
# AB#38607 redux set up for pop-up

## Overview

* Redux set up for pop-up when changing modes. This is associated with ADO task [#38607](https://dev.azure.com/microsoftgarage/Intern%20GitHub/_sprints/taskboard/GI21%20-%20Graph%20Explorer/Intern%20GitHub/Y21-S/06?workitem=38607). 
* Necessary to be done before defining the flag of when the pop up will be shown.
* `CLOSE_POP_UP_SUCCESS` is for when the rsc pop up closes.
* `PREPARE_NEXT_POP_UP_SUCCESS` currently shows up to reset the flag and indicates it will show up the next time the person toggles to app mode again. This will be changed to `OPEN_POP_UP_SUCCESS` and be triggered when the pop-up actually shows up when the flag is implemented. 

### Demo

https://user-images.githubusercontent.com/47228306/125356549-35226200-e31b-11eb-8a37-e108a9d1dab6.mp4

### Notes

![Screenshot 2021-07-12 133930](https://user-images.githubusercontent.com/47228306/125356658-5b480200-e31b-11eb-8941-fd8b000f515d.png)


## Testing Instructions

* `npm install && npm start`
* press f12 to inspect
* toggle between user and application mode to see the changes for the rsc pop up
